pkg/telemetry/telemetry_test.go

```
@@ -0,0 +1,718 @@
+	mockClient.EXPECT().Close().Return(nil).Times(0)
+
+	telemetry := NewTelemetry(enabled, token, endpoint, distinctId, mockClientProvider.NewMockClient)
+
+	assert.Equal(t, telemetry.isEnabled, enabled)
+	assert.Equal(t, telemetry.token, token)
+	assert.Equal(t, telemetry.endpoint, endpoint)
+	assert.Equal(t, telemetry.distinctId, distinctId)
+	assert.NotNil(t, telemetry.clientProvider)
+}
+
+func TestTelemetryCaptureEventMethod(t *testing.T) {
```

golangci-lint (1.64.8)

[error] 58-99 lines are duplicate of `pkg/telemetry/telemetry_test.go:101-142`

(dupl)